pkg/globalconfig/global_config.go

```diff
@@ -627,6 +637,11 @@ func IsInternetActive() bool {
 	return active
 }
 
+// IsTraefikRouter returns true if the router is traefik
+func IsTraefikRouter() bool {
+	return DdevGlobalConfig.Router == globalconfigtypes.RouterTypeTraefik
+}
```

Comment:
```suggestion
func (c *GlobalConfig) IsTraefikRouter() bool {
	return c.Router == globalconfigtypes.RouterTypeTraefik
}
```

Because you access config properties this should be a method instead.